tools/src/main/java/org/apache/kafka/tools/FeatureCommand.java

```diff
@@ -282,6 +298,32 @@ static void handleDisable(Namespace namespace, Admin adminClient) throws TerseEx
         update("disable", adminClient, updates, namespace.getBoolean("dry_run"));
     }
 
+    static void handleVersionMapping(Namespace namespace) throws TerseException {
+        // Get the release version from the command-line arguments or default to the latest stable version
+        String releaseVersion = Optional.ofNullable(namespace.getString("release_version"))
+            .orElseGet(() -> {
+                String latestVersion = MetadataVersion.latestProduction().version();
+                System.out.println("No release version provided. Defaulting to the latest stable version: " + latestVersion);
```

Comment:
Does this tool typically print System.out like this? I think it makes sense for the output, but maybe it is enough to just show the version being used as you do on line 314

Reply:
I think so? the describe command uses printf and the update method uses println

Reply:
I guess I'm just wondering if this line is needed. I think it could make it harder to parse the output if it is expecting a certain format.

Reply:
ahhh I get it now! I'll remove it thanks!